app/bundles/LeadBundle/Form/Type/CustomYesNoButtonGroupType.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Mautic\LeadBundle\Form\Type;
+
+use Mautic\CoreBundle\Form\Type\ButtonGroupType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CustomYesNoButtonGroupType extends AbstractType
```

Comment:
Make all new classes final by default. I think Nullable instead of Custom could define the form type better. What do you think?
```suggestion
final class NullableYesNoButtonGroupType extends AbstractType
```